test-openshift/src/main/java/io/debezium/test/openshift/tools/HttpUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.test.openshift.tools;
+
+import static org.awaitility.Awaitility.await;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Utility methods for HTTP requests
+ * @author Jakub Cechacek
+ */
+public class HttpUtils {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private final OkHttpClient http;
+
+    public HttpUtils(OkHttpClient http) {
+        this.http = http;
+    }
+
+    /**
+     * Waits until URL starts responding with success response code
+     * @param url tested url
+     */
+    public void awaitApi(HttpUrl url) {
+        logger.info("Waiting for API at " + url);
+        await()
+                .atMost(1, TimeUnit.MINUTES)
```

Comment:
Does it make sense to make this (and others) `await()` instance configurable via system properties? Nothing fancy just two different wait parameters - one for short one one for long waits.